specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-05-01-privatepreview/examples/AlertProcessingRules_Create_or_update_add_correlation.json

```diff
@@ -0,0 +1,231 @@
+{
+  "parameters": {
+    "api-version": "2023-05-01-privatepreview",
+    "subscriptionId": "subId1",
+    "resourceGroupName": "alertscorrelationrg",
+    "alertProcessingRuleName": "CorrelationExample",
+    "alertProcessingRule": {
+      "location": "Global",
+      "tags": {},
+      "properties": {
+        "scopes": [
+          "/subscriptions/subId1"
+        ],
+        "actions": [
+          {
+            "actionType": "Correlation",
+            "correlateBy": [
+              {
+                "field": "essentials.alertRuleName",
+                "type": "Simple"
+              },
+              {
+                "field": "essentials.severity",
+                "type": "Grouped",
+                "groups": [
+                  {
+                    "operator": "Equals",
+                    "values":[
+                      "sev0",
+                      "sev1"
+                    ]
+                  },
+                  {
+                    "operator": "Equals",
+                    "values":[
+                      "sev2",
+                      "sev3",
+                      "sev4"
+                    ]
+                  },
+                  {
+                    "field": "context.labes['clusterName']",
+                    "type": "Grouped",
+                    "groups": [
+                      {
+                        "operator": "StartsWith",
+                        "values": [
+                          "Prod"
+                        ]
+                      },
+                      {
+                        "operator": "StartsWith",
+                        "values": [
+                          "Test"
+                        ]
+                      }
+                    ]
+                  }
+                ]
+              }
+            ],
+            "correlationTimeWindow": "00:30:00",
```

Comment:
What are the supported values for "correlationTimeWindow"?

Reply:
it's a time span property - as for min/max values, lets sync offline

Reply:
min/max talk with Harel